app/views/speakers/_form.html.erb

```diff
@@ -11,71 +11,72 @@
     </div>
   <% end %>
 
-  <h2>Speaker</h2>
-
-  <div class="field">
-    <%= form.label :name %>
-    <%= form.text_field :name, id: :speaker_name %>
-  </div>
-
-  <div class="field">
-    <%= form.label :picture %>
-    <%= form.file_field :picture %>
-  </div>
-
-  <div class="field">
-    <%= form.label :email %>
-    <%= form.text_field :email %>
-  </div>
-
-  <div class="field">
-    <%= form.label :company %>
-    <%= form.text_field :company %>
-  </div>
-
-  <div class="field">
-    <%= form.label :biography %>
-    <%= form.text_area :biography %>
-  </div>
+  <div class="form">
+    <h2>Palestrante</h2>
+    <div class="field">
+      <%= form.label :nome %>
+      <%= form.text_field :name, id: :speaker_name, class: :input %>
+    </div>
 
-  <div class="field">
-    <%= form.label :website %>
-    <%= form.text_field :website %>
-  </div>
+    <div class="field">
+      <%= form.label :foto %>
+      <%= form.file_field :picture, class: :inputpicture %>
+    </div>
 
-  <%= form.fields_for :talk, @talk do |talk_form| %>
-    <h2>Talk</h2>
-    
     <div class="field">
-      <%= talk_form.label :title %>
-      <%= talk_form.text_field :title, id: :talk_title %>
+      <%= form.label :email %>
+      <%= form.text_field :email, class: :input %>
     </div>
 
     <div class="field">
-      <%= talk_form.label :description %>
-      <%= talk_form.text_area :description, id: :talk_description %>
+      <%= form.label :empresa %>
+      <%= form.text_field :company, class: :input%>
     </div>
 
     <div class="field">
-      <%= talk_form.label :theme %>
-      <%= talk_form.select(:theme) do %>
-        <% [['Web', :web], ['Games', :games], ['DevOps', :dev_ops], ['Carreira', :carrer], ['Design', :design], ['Mobile', :mobile], ['Análise', :analysis], ['Outros', :other]].each do |theme| -%>
-          <%= content_tag(:option, theme.first, value: theme.last) %>
-        <% end %>
-      <% end %>
+      <%= form.label :biografia %>
+      <%= form.text_area :biography, class: :textarea %>
     </div>
 
     <div class="field">
-      <%= talk_form.label :required_knowledge %>
-      <%= talk_form.select(:required_knowledge) do %>
-        <% [['Iniciante', :beginner], ['Médio', :medium], ['Avançado', :advanced]].each do |theme| -%>
-          <%= content_tag(:option, theme.first, value: theme.last) %>
-        <% end %>
-      <% end %>
+      <%= form.label :website %>
+      <%= form.text_field :website, class: :input%>
     </div>
-  <% end %>
+    <%= form.fields_for :talk, @talk do |talk_form| %>
+      <h2>Talk</h2>
+
+      <div class="field">
+        <%= talk_form.label :titulo %>
```

Comment:
im going to stop say 'same here', but you get the idea. It is valid for all the form fields.